src/librustdoc/html/render/mod.rs

```diff
@@ -1066,7 +1066,7 @@ themePicker.onblur = handleThemeButtonsBlur;
                 krates
                     .iter()
                     .map(|s| {
-                        format!("<li><a href=\"{}index.html\">{}</li>", ensure_trailing_slash(s), s)
+                        format!("<li><a class=\"mod\" href=\"{}index.html\">{}</a></li>", ensure_trailing_slash(s), s)
```

Comment:
Using class `mod` should probably be temporary (I should probably be a separate class `crate`). However, the `ul` that this is in (starting on line 1065) already has class `mod`!